Services/DataBaseService/MigrationScripts/03.2020/02_ScriptsTableCreate.sql

```diff
@@ -0,0 +1,13 @@
+USE [TradingStation];
+
+IF OBJECT_ID('dbo.ExecutedScripts', 'U') IS NOT NULL
+    DROP TABLE [dbo].[ExecutedScripts]
+
+CREATE TABLE [dbo].[ExecutedScripts]
+    (
+        Id	    INT IDENTITY     NOT NULL,
```

Comment:
Why id here is int? in user table it is guid

Reply:
> Why id here is int? in user table it is guid

Since there's no need to prevent bruteforce, and to simplify any reconstruction of events. Am I wrong?